src/blocks.zig

```diff
@@ -256,6 +271,9 @@ pub fn hasRegistered(id: []const u8) bool {
 pub const Block = packed struct { // MARK: Block
 	typ: u16,
 	data: u16,
+
+	pub const Air = Block{.typ = 0, .data = 0};
```

Comment:
This is not a `type`, so it should be lowercase.
```suggestion
	pub const air = Block{.typ = 0, .data = 0};
```